recipes/pysdl2/locate_sdl.patch

```diff
@@ -0,0 +1,18 @@
+--- sdl2/__init__.py
++++ sdl2/__init__.py
+@@ -1,4 +1,15 @@
+ """SDL2 wrapper package"""
++# Direct this module to Ana/Miniconda's lib folder, where the SDL2 libraries
++# can be found.
++import os
++
++if os.name == "posix":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir))
++elif os.name == "nt":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir, "Library", "bin"))
```

Comment:
Alright, I see that on Mac OS X, sys.prefix outputs my anaconda folder, which is fine, because that contains the lib, bin, and include folders. On Windows however, it does the same, but shouldn't the prefix be `Anaconda/Library` there? Simply using the sys.prefix therefore wouldn't work on Windows.